com.unity.ml-agents/Tests/Editor/Unity.ML-Agents.Editor.Tests.asmdef

```diff
@@ -23,5 +23,12 @@
     "autoReferenced": false,
     "defineConstraints": [
         "UNITY_INCLUDE_TESTS"
+    ],
+    "versionDefines": [
+        {
+            "name": "com.unity.modules.unityanalytics",
+            "expression": "1.1.0",
```

Comment:
Was version `1.1.0` by design here? Shouldn't it be `1.0.0`?

Reply:
No, must have missed this one